addons/website/static/src/js/widgets/theme.js

```diff
@@ -541,6 +551,21 @@ var ThemeCustomizeDialog = Dialog.extend({
 
         return def;
     },
+    /**
+     * @private
+     */
+    _setAuto: function ($inputData) {
+        var self = this;
+        var values = {};
+        var isChecked = $($inputData).siblings('.o_theme_customize_option_input').is(':checked');
```

Comment:
- `$($inputData)` ? Why wrapping in jQuery again ?
- Instead of `.is(':checked')` use `.prop('checked')` which is more efficient